lib/network/modules/components/Edge.js

```diff
@@ -510,7 +534,12 @@ class Edge {
     this.drawLabel (ctx, viaNode);
   }
 
-
+  /**
+   *
+   * @param {CanvasRenderingContext2D} ctx
+   * @param {Object} arrowData
+   * @param {{toArrow: boolean, toArrowScale: (allOptions.edges.arrows.to.scaleFactor|{number}|allOptions.edges.arrows.middle.scaleFactor|allOptions.edges.arrows.from.scaleFactor|Array|number), toArrowType: *, middleArrow: boolean, middleArrowScale: (number|allOptions.edges.arrows.middle.scaleFactor|{number}|Array), middleArrowType: (allOptions.edges.arrows.middle.type|{string}|string|*), fromArrow: boolean, fromArrowScale: (allOptions.edges.arrows.to.scaleFactor|{number}|allOptions.edges.arrows.middle.scaleFactor|allOptions.edges.arrows.from.scaleFactor|Array|number), fromArrowType: *, arrowStrikethrough: (*|boolean|allOptions.edges.arrowStrikethrough|{boolean}), color: undefined, inheritsColor: (string|string|string|allOptions.edges.color.inherit|{string, boolean}|Array|*), opacity: *, hidden: *, length: *, shadow: *, shadowColor: *, shadowSize: *, shadowX: *, shadowY: *, dashes: (*|boolean|Array|allOptions.edges.dashes|{boolean, array}), width: *}} values
```

Comment:
What it does add is auto-complete in IDEs when working with the code, and potentially better ts generation.

I agree, it is not ideal, and it probably makes more sense to make `getFormattingValues` return a proper class that is well documented, and then anything using `values` can just use the class.